client/state/data-layer/wpcom/jetpack-onboarding/index.js

```diff
@@ -12,9 +12,63 @@ import { translate } from 'i18n-calypso';
 import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
 import { errorNotice } from 'state/notices/actions';
 import { http } from 'state/data-layer/wpcom-http/actions';
-import { JETPACK_ONBOARDING_SETTINGS_SAVE } from 'state/action-types';
+import {
+	JETPACK_ONBOARDING_SETTINGS_REQUEST,
+	JETPACK_ONBOARDING_SETTINGS_SAVE,
+} from 'state/action-types';
+import { getUnconnectedSite } from 'state/selectors';
 import { updateJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions';
 
+export const fromApi = response => {
+	if ( ! response.data.onboarding || ! response.data.onboarding ) {
```

Comment:
Did you mean `! response.data || ! response.data.onboarding`? An additional `fromApi` test could have caught this, so maybe you'd like to add one?